packages/serverless-nextjs-component/next-aws-cloudfront.js

```diff
@@ -125,14 +126,32 @@ const handler = event => {
       Buffer.isBuffer(chunk) ? chunk : Buffer.from(chunk)
     ]);
   };
+  let gz = false;
+  const ae = headers["accept-encoding"];
```

Comment:
Can this be refactored into it's own function?

Reply:
yes definitely

Reply:
Hey, @barrysteyn I have done required changes.